addons/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/DSMRBindingConstants.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.dsmr;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link DSMRBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author M. Volaart
+ * @since 2.1.0
+ */
+public class DSMRBindingConstants {
+    /**
+     * Binding constant
+     */
+    public static final String BINDING_ID = "dsmr";
+
+    /**
+     * Constant for the DSMR Thread pool
+     */
+    public static final String DSMR_THREAD_POOL_NAME = "DSMR Thread Pool";
```

Comment:
Normally, bindings must not create their own thread pool, but reuse the existing "scheduler".
Do we have a special situation here which requires us to have a separate pool?